### PR TITLE
Edits to PR4538 database administration for epas

### PR DESCRIPTION
**Pull request by ebgitelman**
First pass

### CODE REVIEW

File: product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx [hunk 1]
```diff
@@ -21,9 +21,9 @@ EDB\*Loader features include:
 
 The important version compatibility restrictions between the EDB\*Loader client and the database server are:
 
--   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version 14 EDB\*Loader client (the edbldr program supplied with EDB Postgres Advanced Server 14) to load data only into version 14 of the database server. In general, use the same version for the EDB\*Loader client and database server.
+-   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version 14 EDB\*Loader client (the `edbldr` program supplied with EDB Postgres Advanced Server 14) to load data only into version 14 of the database server. In general, use the same version for the EDB\*Loader client and database server.
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx:24]:
```suggestion
-   When you invoke the EDB\*Loader program (called `edbldr`), you pass in parameters and directive information to the database server. We strongly recommend that you use the version of EDB\*Loader client, the `edbldr` program, supplied with the version of EDB Postgres Advanced Server you are using to load data only into the database server. In general, use the same version for the EDB\*Loader client and database server.
```

Comment (drothery-edb) [reply product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx:24]: @nidhibhammar is there anything special about v14? I doubt it. I think this is just legacy content that got rolled over. I think we can make this version agnostic.

**Review by drothery-edb — DISMISSED** [review]

File: product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx:24]: @tusharahuja, is there anything specific about the edb loader v14 client?

Comment (tusharahuja) [reply product_docs/docs/epas/15/database_administration/02_edb_loader/edb_loader_overview_and_restrictions.mdx:24]: Right, there is nothing particular about edb*loader v14. Every EPAS server version comes with edb*loader which is available under the bin folder. Thanks.

**Review by nidhibhammar — APPROVED** [review]

**Review by francoughlin — APPROVED** [review]
Looks good!

**Pull request merged**